.github/workflows/opentelemetry-e2e.yml

```diff
@@ -0,0 +1,30 @@
+name: Opentelemetry e2e
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  e2e:
+    name: Opentelemetry e2e
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
```

Comment:
Fetch depth 1? Missing git fetch github.base_ref? Why is the value set to what it is?